src/services/symbolDisplay.ts

```diff
@@ -836,6 +837,14 @@ export function getSymbolDisplayPartsDocumentationAndSymbolKind(typeChecker: Typ
             tags = allSignatures[0].getJsDocTags().filter(tag => tag.name !== "deprecated"); // should only include @deprecated JSDoc tag on the first overload (#49368)
         }
 
+        if (symbolFlags & SymbolFlags.Alias && documentation.length === 0 && tags.length === 0) {

```

Comment:
I think it would be better to put this code inside the existing `if (symbolFlags & SymbolFlags.Alias)` block around line 536. It already sets documentationFromAlias/tagsFromAlias under other conditions, so it would be a good idea to make sure they all work together.

Reply:
What I ended up with was:

```ts
    if (symbol.flags & SymbolFlags.Alias) {
        prefixNextMeaning();
        const noDocumentation = documentation.length === 0 && tags.length === 0
        if (!hasAddedSymbolInfo || noDocumentation) {
            const resolvedSymbol = typeChecker.getAliasedSymbol(symbol);
            if (resolvedSymbol !== symbol && resolvedSymbol.declarations && resolvedSymbol.declarations.length > 0) {
                const resolvedNode = resolvedSymbol.declarations[0];
                const declarationName = getNameOfDeclaration(resolvedNode);
                if (noDocumentation) {
                    documentationFromAlias = resolvedSymbol.getContextualDocumentationComment(resolvedNode, typeChecker);
                    tagsFromAlias = resolvedSymbol.getJsDocTags(typeChecker);
                }
                else if (declarationName) {
```

from what was formerly

```ts
    if (symbol.flags & SymbolFlags.Alias) {
        prefixNextMeaning();
        if (!hasAddedSymbolInfo || noDocumentation) {
            const resolvedSymbol = typeChecker.getAliasedSymbol(symbol);
            if (resolvedSymbol !== symbol && resolvedSymbol.declarations && resolvedSymbol.declarations.length > 0) {
                const resolvedNode = resolvedSymbol.declarations[0];
                const declarationName = getNameOfDeclaration(resolvedNode);
                if (declarationName) {
```